Subject: thermal: exynos: Fix wrong name of MODULE_DEVICE_TABLE
Date: Mon, 22 Oct 2012 11:19:10 -0000
From: jhbird.choi@samsung.com
X-Patchwork-Id: 1625241
Patch-mainline: no, committed as https://patchwork.kernel.org/patch/1625241/

exynos4_tmu_driver_ids should be exynos_tmu_driver_ids.

Signed-off-by: Jonghwan Choi <jhbird.choi@samsung.com>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
drivers/thermal/exynos_thermal.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

                        struct platform_device *pdev)
--
1.7.1

--
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/

Index: linux-3.7-rc2-master/drivers/thermal/exynos_thermal.c
===================================================================
--- linux-3.7-rc2-master.orig/drivers/thermal/exynos_thermal.c
+++ linux-3.7-rc2-master/drivers/thermal/exynos_thermal.c
@@ -815,7 +815,7 @@ static struct platform_device_id exynos_
 	},
 	{ },
 };
-MODULE_DEVICE_TABLE(platform, exynos4_tmu_driver_ids);
+MODULE_DEVICE_TABLE(platform, exynos_tmu_driver_ids);
 
 static inline struct  exynos_tmu_platform_data *exynos_get_driver_data(
 			struct platform_device *pdev)
